ffmpeg/build_ci.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+LATEST_COMMIT=$(git rev-parse HEAD)
+FOLDER_COMMIT=$(git log -1 --format=format:%H --full-diff $DIR)
+
+if [ $FOLDER_COMMIT = $LATEST_COMMIT ]; then
+    echo "FFMpeg has changed"
+    ./$DIR/build_ffmpeg.sh
+
+    echo "Uploading FFmpeg to PyTorch Anaconda channel"
+    anaconda upload -u pytorch --force $DIR/output/*/ffmpeg*.bz2
```

Comment:
We should probably just split this out into it's own job so we can have separate build / upload jobs.

This is to make it easier to test any changes as they come in.

Reply:
You can persist workspace artifiacts using the `persist_to_workspace` functionality within circleci: https://circleci.com/docs/2.0/configuration-reference/#persist_to_workspace